packages/hint-no-vulnerable-javascript-libraries/src/hint.ts

```diff
@@ -174,7 +174,7 @@ export default class NoVulnerableJavascriptLibrariesHint implements IHint {
                 context.report(resource, message, {
                     documentation: [{
                         link,
-                        text: getMessage('learnMore', context.language)
+                        text: getMessage('learnMore', context.language, [vulnerabilities[0].id, vulnerabilities[0].severity])
```

Comment:
We should loop the `vulnerabilities` collection so we have one link per vulnerability.